docs/cloud-messaging/receive.md

```diff
@@ -273,3 +273,63 @@ class _Application extends State<Application> {
 ```
 
 How you handle interaction depends on your application setup. The above example shows a basic illustration using a StatefulWidget.
+
+## Localize messages
+Localizing notification is embedded in the underlying system (Android or iOS). You can send localized
+strings 2 ways:
+* Storing the language of your user in your server and send customized notification for each language
+* Add native locale strings to your app
+
+For the second method, you can check how to use it below: 
+### Android
+
+1. resources/values/strings.xml (default file)
+
+```xml
+<string name="notification_title">Hello world</string>
+<string name="notification_message">This is a message</string>
+```
+2. resources/values-fr/strings.xml (French strings file)
+
+```xml
+<string name="notification_title">Bonjour le monde</string>
+<string name="notification_message">C'est un message</string>
+```
+On the server payload instead of using **title** and **message** / **body**  keys, we will use **title_loc_key** and **body_loc_key** for our localized message.
```

Comment:
```suggestion
On the server payload, instead of using **title** and **message** / **body**  keys, we will use **title_loc_key** and **body_loc_key** for our localized message.
```